hikari/impl/special_endpoints.py

```diff
@@ -1253,6 +1253,20 @@ def build(
         return {"type": self.type, "data": data}, ()
 
 
+@attrs.define(kw_only=False, weakref_slot=False)
+class InteractionPremiumRequiredBuilder(special_endpoints.InteractionPremiumRequiredBuilder):
+    """Standard implementation of `hikari.api.special_endpoints.InteractionPremiumRequiredBuilder`."""
+
+    @property
+    def type(self) -> typing.Literal[base_interactions.ResponseType.PREMIUM_REQUIRED]:
+        return base_interactions.ResponseType.PREMIUM_REQUIRED
+
+    def build(
+        self, entity_factory: entity_factory_.EntityFactory, /
+    ) -> typing.Tuple[typing.MutableMapping[str, typing.Any], typing.Sequence[files.Resource[files.AsyncReader]]]:
+        return {"type": self.type, "data": {}}, ()
```

Comment:
Do we need to send an empty data?

Reply:
Unsure, this is how the API docs has it though
https://discord.com/developers/docs/monetization/entitlements#premiumrequired-interaction-response

Reply:
Data can be not sent for some interactions, so maybe worth testing

Reply:
~~Now I just need a bot that can send these~~ :p

Reply:
Turns out its not needed